src/tests/ftest/io/eight_servers.yaml

```diff
@@ -22,50 +22,52 @@ server_config:
     name: daos_server
 pool:
     createmode:
-        mode_RW: 146
+        mode_RW:
+            mode: 146
     createset:
         setname: daos_server
     createsize:
-        size: 60000000000
+        scm_size: 60000000000
     createsvc:
         svcn: 1
 ior:
-    clientslots: !mux
+    client_processes: !mux
         np_1:
             np: 1
         np_64:
             np: 64
         np_128:
             np: 128
-    iteration:
-        iter: 5
     iorflags:
         sequential:
             F: "-v -W -w -r -R"
         random:
             F: "-v -z -W -w -r -R"
+    api: DAOS
+    test_file: daos:testFile
+    repetitions: 5
     transfersize_blocksize: !mux
         1K:
-            t: '1K'
-            b: '32M'
+            transfer_size: '1K'
+            block_size: '32M'
         4K:
-            t: '4K'
-            b: '64M'
+            transfer_size: '4K'
+            block_size: '64M'
         32K:
-            t: '32K'
-            b: '64M'
+            transfer_size: '32K'
+            block_size: '64M'
         128K:
-            t: '128K'
-            b: '64M'
+            transfer_size: '128K'
+            block_size: '64M'
         512K:
-            t: '512K'
-            b: '64M'
+            transfer_size: '512K'
+            block_size: '64M'
         1M:
-            t: '1M'
-            b: '64M'
+            transfer_size: '1M'
+            block_size: '64M'
     objectclass: !mux
-        LARGE:
-            o: "LARGE"
+        SX:
+            daos_oclass: "SX"
         2-way_Replication:
-            o: "R2"
+            daos_oclass: "RP_2G2"
```

Comment:
Why not use the "RP_2GX" , this will use all available targets. I think this is what R2 was doing.

Reply:
Fixed